pkgs/games/steam/fhsenv.nix

```diff
@@ -279,7 +285,16 @@ in buildFHSEnv rec {
     exec steam ${extraArgs} "$@"
   '';
 
-  inherit extraBwrapArgs;
+  # steamwebhelper deletes unrelated electron programs' singleton cookies from /tmp on startup
+  privateTmp = true;
```

Comment:
Nit: Mention the upstream issue here

Reply:
It is already in the commit message, which can be seen through git blame, but if you really want me to put it in the comment, too, I can.

Reply:
Commit messages are for backgrounds/motivations for the commit's changes.  
Code comments are for extended info on certain pieces of unexpected or unintuitive code.

If someone came around in the future and sees this, they need to be able to verify the bug still exists and therefore need the upstream issue link. They shouldn't have to dig that out of a blame in the first place but additionally, the blame may no longer even point to your original commit as there may have been multiple refactors in the mean time.